backend/src/rss.rs

```diff
@@ -216,6 +233,9 @@ async fn video_items(
         let event_link = format!("{tobira_url}/!v/{tobira_event_id}");
         let thumbnail = &event.thumbnail_url.unwrap_or_default();
         let creators = event.creators.join(", ");
+        let event_author = extract_author(&event.metadata)
+            .or_else(|| event.creators.first().map(|s| s.to_string()))
+            .unwrap_or_default();
```

Comment:
Why default to empty string here, but omit the whole tag for the series if there is no author? If there are no reasons for this, probably should make it consistent with my preference being to omit the tag like for series.